src/Microsoft.VisualStudio.Composition/ReflectionHelpers.cs

```diff
@@ -106,22 +111,16 @@ internal static Assignability IsAssignableTo(ImportDefinitionBinding import, Exp
             }
             else
             {
-                // Utilize the standard assignability checks for everything else.
-                if (receivingType.GetTypeInfo().IsAssignableFrom(exportingType.GetTypeInfo()))
-                {
-                    return Assignability.Definitely;
-                }
-
                 bool valueTypeKnownExactly =
                     export.ExportingMemberRef == null || // When [Export] appears on the type itself, we instantiate that exact type.
-                    exportingType.GetTypeInfo().IsSealed;
+                    exportingType.ResolvedType.GetTypeInfo().IsSealed;
```

Comment:
Of the remaining `ResolvedType` uses, what % of assemblies end up getting loaded after the changes you've already made? What % of assemblies were loaded before your changes?